t/sanity.t

```diff
@@ -944,7 +944,7 @@ healthcheck: peer 127.0.0.1:12359 is turned down after 2 failure(s)
 qr/^(?:spawn a thread checking primary peers 0 to 2
 check primary peers 3 to 4
 check backup peer 0
-){4,6}$/
+){4,8}$/
```

Comment:
These changes are irrelevant. Are you sure you are running the latest git master version of test-nginx?